program/src/oracle/util/align.h

```diff
@@ -0,0 +1,57 @@
+#ifndef _pyth_oracle_util_align_h_
+#define _pyth_oracle_util_align_h_
+
+#include "compat_stdint.h" /* For uintptr_t */
+#include <stdalign.h>      /* For alignof */
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
```

Comment:
my brain hurts at this point. i'm assuming these utils are fine. I checked avg carefully. the rest of these seem mechanical and also thoroughly tested.